packages/compiler/server/serverlib.ts

```diff
@@ -639,20 +647,70 @@ export function createServer(host: ServerHost): Server {
 
   /**
    * Get the detailed documentation of a type.
+   * @param program The program
    */
   function getTypeDetails(program: Program, type: Type): string {
+    // TODO: Should not use only resolved type here so we can show alias docs and definitions.
```

Comment:
todo

Reply:
Filed #1348 and replaced comment with link to that. I'd like to do some refactoring to fix this.